website/sidebars.json

```diff
@@ -51,7 +54,85 @@
         ]
       }
     ],
-    "aragonUI": ["aragonui-intro"],
-    "App Center": ["app-center-preparing-assets"]
+    "aragonUI": [
+      "aragonui-intro"
+    ],
+    "App Center": [
+      "app-center-preparing-assets"
+    ]
+  },
+  "docs-api": {
```

Comment:
Now that we have nested sidebars, would it be possible to add this into an "API" section in aragonOS?

It might also be a good time to move the aragonOS 3 docs into their own nested sidebar under aragonOS.